### DISCUSSION TITLE
MVC/Tiered Application Service Issue

### DISCUSSION THREAD

**Discussion by spospisil** [abpframework/abp#12413] [Q&A]
I have a project where I add a new application services that inherits from 'ApplicationService' and implements an 'interface' that inherits from the 'IApplicationService' interface.

When I run the project it tells me that this service is not registered.  Wondering what else needs to be done or what else I can check for in order to be able to use/inject this service in my code.

**Comment by maliming** [comment]
Please share some of your code.

**Comment by ashkandehnavi** [comment]
Have you read the [dependency injection](https://docs.abp.io/en/abp/latest/Dependency-Injection) document?
